src/sass/base.scss

```diff
@@ -24,9 +24,7 @@
 /**
  * Styles shared between the different apps (mobile and desktop).
  */
-@import '~ol/ol.css';
-@import '~gmf/css/reset.css';
-@import '~bootstrap/scss/bootstrap.scss';
+@import 'gmf/sass/vars_only.scss';
```

Comment:
Do you remember why you made those changes?

Reply:
These three deleted line are moved into `src/controllers/desktop.scss` and `src/controllers/mobile.scss`

Yes, it's because `gmf/sass/gmf-font.scss` (with fontawesome loading) must be directly above bootstrap, and I didn't wanted to add `gmf-font.scss` into the `base.scss` file.
Also, It's not so bad to let the possibility to import or not these "basique" (but quite huge!) scss files.

Reply:
OK, thanks for the explain :-)